packages/contracts-bedrock/src/libraries/DisputeErrors.sol

```diff
@@ -16,7 +16,7 @@ error NoImplementation(GameType gameType);
 error GameAlreadyExists(Hash uuid);
 
 ////////////////////////////////////////////////////////////////
-//               `DisputeGame_Fault.sol` Errors               //
+//               `FaultDisputeGame.sol` Errors                //
```

Comment:
I think we should remove the `.sol` here - i think it looks ugly and illegible

Reply:
Let's do this in another PR.